pyre/extra_tests/parity_tests/warnoptions_category_module_raises.py

```
@@ -0,0 +1,41 @@
+with tempfile.TemporaryDirectory() as tmp:
+    with open(os.path.join(tmp, "raising_category.py"), "w") as module:
+        module.write('raise RuntimeError("raised-from-the-category-module")\n')
+    env = dict(os.environ)
+    env["PYTHONPATH"] = tmp + os.pathsep + env.get("PYTHONPATH", "")
+    result = subprocess.run(
+        [sys.executable, "-W", "ignore::raising_category.W", "-c", "print('RAN')"],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        check=False,
+        env=env,
+    )
```

Ruff (0.16.2)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)